docs/README.md

```diff
@@ -1,6 +1,11 @@
-# Duke
+# Upcycle
 
-{Give product intro here}
+Hi there, 
+
+Welcome to Upcycle - a perfect desktop app dedicate for managing rental businesses. The nature of these businesses
+requires to deal with tons of data; therefore, noting down all information by notes and pen is not ideal for managers.
+It is developed for rental business managers, who can type fast to efficiently keep track all of their customers, items,
+and transactions via Command Line Interface with a single line of command.
 
 Useful links:
 * [User Guide](UserGuide.md)
```

Comment:
Activation box ought to not immediately be closed, should leave some time for clearing the object up
![1](https://user-images.githubusercontent.com/71815678/198494584-98460458-1e5e-40b5-8b7b-da6a8f911072.png)